cba_settings.sqf

```diff
@@ -781,7 +781,7 @@ force force kat_circulation_useLocation_AED = 0;
 
 // KAT - ADV Medical: GUI
 kat_gui_ColoredLogs = true;
-kat_gui_overlayBodyPart = true;
+force force kat_gui_overlayBodyPart = true;
 force force kat_gui_showBleedRate = false;
 force force kat_gui_showInactiveStatuses = true;
```

Comment:
Overall i had a policy on on having user peferences on all GUI client only stuff.
```suggestion
//kat_gui_overlayBodyPart = true;
//kat_gui_showBleedRate = false;
//kat_gui_showInactiveStatuses = true;
```

Reply:
These don't fall in that category. The medical system relies heavily on the body part and the bleed rate is a game play issue along with overlay and the inactive statuses. We have those set purposefully